src/data/polynomial/basic.lean

```diff
@@ -582,6 +591,10 @@ begin
   { rw [pow_succ', hn, X, monomial_mul_monomial, one_mul] },
 end
 
+@[simp] lemma to_finsupp_X_pow {R : Type u} [semiring R] (n : ℕ) :
+  (X ^ n).to_finsupp = finsupp.single n (1 : R) :=
```

Comment:
Unless I'm missing something,
```suggestion
@[simp] lemma to_finsupp_X_pow (n : ℕ) : (X ^ n).to_finsupp = finsupp.single n (1 : R) :=
```
and same throughout

Reply:
I copied them over by mistake... thanks